.github/workflows/build_and_test.yml

```diff
@@ -50,9 +50,15 @@ jobs:
     steps:
       - name: Checkout repository
         uses: actions/checkout@v4
+      - name: Set up JDK 17
```

Comment:
`ubuntu-latest`'s default JDK is still `Java 11`.

- https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2204-Readme.md#java